internal/pkg/global/config.go

```diff
@@ -476,3 +497,13 @@ func GenerateConfigFromTemplate(templatePath, configPath string, config interfac
 
 	return t.Execute(configFile, config)
 }
+
+// Enabled checks if exporting to the metrika platform is Enabled
+// in agent's configuration.
+// Default: true.
+func (p *PlatformConfig) Enabled() bool {
```

Comment:
Helper's name should be `IsEnabled()` since `platform.enabled` is already reserved.

Reply:
Addressed in 6d05b87